shoop/xtheme/view_config.py

```diff
@@ -21,7 +21,7 @@ class ViewConfig(object):
     container for `SavedViewConfig` objects, and wraps the `SavedViewConfig` API.
     """
 
-    def __init__(self, theme, view_name, draft):
+    def __init__(self, theme, view_name, draft, global_type=False):
```

Comment:
You still have global_type parameter here, but you don't use it in the rendering and editor? I think you probably should keep this and do the view_name override here. So if this init is called with global_type=True then the view_name is overriden into XTHEME_GLOBAL_VIEW_NAME.